modules/types/src/bounds-types.ts

```diff
@@ -0,0 +1,29 @@
+// math.gl
+// SPDX-License-Identifier: MIT
+// Copyright (c) vis.gl contributors
+
+/** 2 dimensional bounds [[minX, minY], [maxX, maxY]] */
+export type Bounds2D = [[number, number], [number, number]];
+
+/** 3 dimensional bounds [[minX, minY, minZ], [maxX, maxY, maxZ]] */
+export type Bounds3D = [[number, number, number], [number, number, number]];
+
+/** 2 or 3 dimensional bounds [[minX, minY], [maxX, maxY]] or [[minX, minY, minZ], [maxX, maxY, maxZ]] */
+export type Bounds =
```

Comment:
Why not `type Bounds = Bounds2D | Bounds3D`?

Reply:
> Why not type Bounds = Bounds2D | Bounds3D?

Yes, this is due to one of my personal preferences. I really like being able to see the actual type definition when I hover over a type in vscode.

By referencing intermediary types, things get obscured and I need to chase down the definition of those types, so if it is easy to type it out I usually do.

But the trend is towards ever more sophisticated typescript types so I am probably on the losing side of history here.